### DISCUSSION TITLE
Missing add group button

### DISCUSSION THREAD

**Discussion by r1si** [GitbookIO/community#221] [0. General bugs and questions]
### Ask a question, start a conversation, or make an annoucement

Hi... I had open a new space yesterday.

In this space missing the ability to create a group of pages...

There is some bug?

**Comment by aikoii** [comment]
Hi @r1si!

Unless I'm missing something, you should be able to add a New Group page like so:

![CleanShot 2023-06-07 at 13 00 13](https://github.com/GitbookIO/community/assets/106068994/defe4de3-7b54-425f-8b5b-a9d76120a14b)

If you still have issues, could you email us at [support@gitbook.com](mailto:support@gitbook.com)? Please include the URL of the space you are working with in the email where you see this happening.

Thank you!
